src/main/java/gyro/google/compute/InstanceNetworkInterface.java

```diff
@@ -150,14 +190,14 @@ public NetworkInterface copyTo() {
             networkInterface.setSubnetwork(getSubnetwork().getSelfLink());
         }
 
-        if (getAccessConfigs() != null) {
-            networkInterface.setAccessConfigs(getAccessConfigs().stream()
+        if (getAccessConfig() != null) {
+            networkInterface.setAccessConfigs(getAccessConfig().stream()
                 .map(InstanceAccessConfig::copyTo)
                 .collect(Collectors.toList()));
         }
 
-        if (getAliasIpRanges() != null) {
-            networkInterface.setAliasIpRanges(getAliasIpRanges().stream()
+        if (getAliasIpRange() != null) {
```

Comment:
If the getter changes are made
```suggestion
        if (!getAliasIpRange().isEmpty()) {
```